src/app/beer_garden/local_plugins/manager.py

```diff
@@ -61,9 +66,11 @@ def start_plugin(self, plugin):
             self.registry.remove(plugin.unique_name)
             self.registry.register_plugin(new_plugin)
         else:
-            raise PluginStartupError("Plugin in an invalid state (%s)" % plugin.status)
+            raise PluginStartupError("Plugin in an invalid state (%s)" % plugin_status)
+
+        plugin_instance.status = "STARTING"
+        update(plugin_instance)
```

Comment:
So to be clear you mean instead of doing this:

```python
from beer_garden.db.api import update

update(plugin_instance)
```

We should do this:
```python
import beer_garden.db.api as db

db.update(plugin_instance)
```

Reply:
This is done in 453a765